pincer/commands/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright Pincer 2021-Present
+# Full MIT License can be found in `LICENSE` at the project root.
+
+from .commands import command, user_command, message_command, ChatCommandHandler
+from .arg_types import (
+    CommandArg, Description, Choice, Choices, ChannelTypes, MaxValue, MinValue
+)
+
+__all__ = [
+    "command", "ChatCommandHandler", "CommandArg", "Description", "Choice",
+    "Choices", "ChannelTypes", "MaxValue", "MinValue", "user_command",
+    "message_command"
+]
```

Comment:
Change this into a tuple. IDK if functionally it affects anything but it will be more consistent with the rest of Pincer.